go/cmd/api/main.go

```diff
@@ -132,13 +132,45 @@ var Cmd = &cli.Command{
 			Value:    0,
 			Required: false,
 		},
+
+		// Vault Configuration
 		&cli.StringSliceFlag{
 			Name:     "vault-master-keys",
 			Usage:    "Vault master keys for encryption",
 			Sources:  cli.EnvVars("UNKEY_VAULT_MASTER_KEYS"),
 			Value:    []string{},
 			Required: false,
 		},
+
+		// S3 Configuration
+		&cli.StringFlag{
+			Name:     "s3-url",
+			Usage:    "S3 Compatible Endpoint URL ",
+			Sources:  cli.EnvVars("UNKEY_S3_URL"),
+			Value:    "",
+			Required: false,
+		},
+		&cli.StringFlag{
+			Name:     "s3-bucket",
+			Usage:    "S3 bucket name",
+			Sources:  cli.EnvVars("UNKEY_S3_BUCKET"),
```

Comment:
this one needs to be renamed to include 'vault' in its name, we'll have more s3 buckets soon with deploy and need to differentiate

Reply:
Then we should make it possible to have different access credentials per bucket too right?

Reply:
probably, I'm actuall not sure if you can have multiple buckets per credentials?

but branding all of these as vault credentials is probably best


fwiw long term I think we'll move vault off s3 and into mysql or pg. It was kind of dumb to do in s3